#### Data statistics
- Ensembl **96** / Ensembl Genomes **43** / WormBase ParaSite **12** gene annotations.   
- This release contains **98** single cell RNA-Seq studies, consisting of **881,813** cells, of which **512,172** passed
  our QC from **9** different species.

#### New experiments
- [Malaria Cell Atlas: Single cell transcriptomics of malaria parasites](https://www.ebi.ac.uk/gxa/sc/experiments/E-CURD-2)
- [Single-cell RNA-seq analysis of 20 organs and tissues from individual mice creates a Tabula muris](https://www.ebi.ac.uk/gxa/sc/experiments/E-ENAD-15)
- [Single cell RNA-seq before and after salt treatment in Saccharomyces cerevisiae](https://www.ebi.ac.uk/gxa/sc/experiments/E-GEOD-102475)
- [Single cell RNA-seq of circulating breast cancer cells from 19 women with ER+/HER2- primary tumours](https://www.ebi.ac.uk/gxa/sc/experiments/E-GEOD-75367)
- [Single cell RNA-seq of cancer stem cells from patients with chronic myeloid leukemia during the disease course](https://www.ebi.ac.uk/gxa/sc/experiments/E-GEOD-76312)
- [Single cell RNA-seq of human pancreatic endocrine cells from Juvenile](https://www.ebi.ac.uk/gxa/sc/experiments/E-GEOD-83139)
- [The Global Molecular Landscape of dormant Hematopoietic Stem Cells](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-4547)
- [Single-cell RNA sequencing of spleen-derived LCK-expressing cells from adult zebrafish](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-4617)
- [Lineage-dependent spatial and functional organisation of the mammalian enteric nervous system](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-5553)
- [Deciphering the relationship between polycomb repression and stochastic gene expression from single-cell RNA-seq data](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-5661)
- [Single-cell RNA-seq of murine medullary thymic epithelial cells (mTECs)](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-5727)
- [scRNA-seq of mouse adipose stem and progenitor cells](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-5802)
- [Single-cell RNA sequencing of OT-I CD8+ T cells after stimulation with different affinity ligands](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-6051)
- [Single cell RNA-seq of isolated human foetal L/M-opsin cone photoreceptor populations labelled using the AAV2/9.pR2.1.GFP reporter](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-6058)
- [Single cell RNA sequencing of stem cell-derived retinal ganglion cells](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-6108)
- [Single cell RNA-Seq of the murine non-myocyte cardiac cellulome](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-6173)
- [RNA-sequencing for functional prediction of RNAs](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-6385)
- [Single cell RNA-sequencing of PBMC derived B cells from human healthy donor](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-6386)
- [Interferon gene therapy reprograms the leukemia microenvironment inducing protective immunity to multiple tumor antigens](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-6487)
- [Monocyte-derived IL-1 and IL-6 are differentially required for cytokine release syndrome and neurotoxicity by CAR-T cells.](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-6505)
- [Single cell sequencing of 3 lung carcinomas](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-6653)
- [Reconstructing the human first trimester fetal-maternal interface using single cell transcriptomics - Smartseq 2 data](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-6678)
- [Reconstructing the human first trimester fetal-maternal interface using single cell transcriptomics - 10x data](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-6701)
- [Single cell RNA-sequencing of induced Neural Plate Border Stem Cells](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-6911)
- [Single cell RNA-sequencing of E8.5 stage embryonic neural tissue from mouse](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-6912)
- [Single cell RNA-sequencing of primary mouse Neural Plate Border Stem Cells](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-6925)
- [Single-cell analysis of T-cells from mouse neonate thymuses in a wild type control and a placental specific isoform of Igf2 knock-down (igf2-P0)](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-6945)
- [Single-cell RNA sequencing of mouse germ cells](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-6946)
- [Single-cell RNA-seq of mouse endothelial cells from three distinct embryonic locations](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-6970)
- [Single-cell RNA sequencing of self-maintaining lamina propria macrophages](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-6976)
- [Single-cell RNA sequencing of VE-Cadherin+ cells isolated from mouse E10.5 aorta-gonad-mesonephros region.](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-6987)
- [Genomic encoding of transcriptional burst kinetics](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-7098)
- [Single-cell transcriptional analysis reveals innate lymphoid cell (ILC)-like cells in zebrafish](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-7117)
- [Drop-seq analysis of bleomycin-treated and control mouse lung single-cell suspensions](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-7142)
- [Single cell transcriptional profiling of aortic endothelium identifies a hierarchy from endovascular progenitors to differentiated cells](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-7149)
- [Single-cell transcriptional analysis reveals ILC-like cells in zebrafish](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-7159)
- [Single-cell RNA-seq of mouse developing spinal cord](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-7320)
- [Single-cell RNAseq of control chimeric mouse embryos at embryonic days 7.5 and 8.5 of mouse development](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-7324)
- [Single-cell RNAseq of Tal1 knockout chimeric mouse embryos at embryonic day 8.5 of mouse development](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-7325)
- [Single-cell RNA-seq of Pdgfra+/Sca1+/Cd31- mouse cardiac cells](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-7365)
- [Single-cell RNA-seq of mouse cardiac interstitial cells 3 & 7 days after sham or myocardial infarction injury](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-7376)
- [Single cell RNA sequencing of putative AIRE+ dendritic cells from human paediatric tonsillitis](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-7381)
- [Single cell RNA-seq of mouse skin cells](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-7417)
- [Single-cell RNA sequencing reveals a dynamic stromal niche within the evolving tumour microenvironment](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-7427)
- [Single cell RNA seq of influenza tetramer specific CD8+ T cells from baseline](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-7606)
- [Fibroblastic reticular cells in the spleen - total splenic Stroma cells - single cell RNAseq dataset 2](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-7703)


#### New features
- Improvements to the marker gene heatmap which displays the top 5 marker genes (sorted by p-value) grouped by cluster is available in all experiments. For instance, have a look at the heatmap in:
  - [Melanoma infiltration of stromal and immune cells ](https://www.ebi.ac.uk/gxa/sc/experiments/E-EHCA-2/results/marker-genes)
  - [Single cell transcriptomics of mixed asexual and sexual blood stage parasites of the rodent malaria model *Plasmodium berghei*](https://www.ebi.ac.uk/gxa/sc/experiments/E-ENAD-16/results/marker-genes)
- A revised analysis pipelines which handles both SmartSeq and droplet technologies:
  - RNA-Seq alignment and quantification handled by [Alevin](https://salmon.readthedocs.io/en/latest/alevin.html). The alignment and quantification pipeline is written in NextFlow:
      - [Droplet quantification workflow](https://github.com/ebi-gene-expression-group/scxa-droplet-quantification-workflow/tree/dd1e3e021ccc08d458928cce93f4f5f0a76284e0).
      - [SmartSeq quantification workflow](https://github.com/ebi-gene-expression-group/scxa-smartseq-quantification-workflow/tree/9fb5a1b3eddf3e236a2b140d3a841366df31fa67)
  - Filtering and clustering handled by [Scanpy](). The filtering and clustering pipelines are written in Galaxy:
      - [Droplet filtering and clustering](https://raw.githubusercontent.com/ebi-gene-expression-group/scxa-workflows/23d2ce059e3b2d00b1ca9fc2806280527bad31ae/w_smart-seq_clustering/scanpy_clustering_workflow.json)
      - [SmartSeq filtering and clustering](https://raw.githubusercontent.com/ebi-gene-expression-group/scxa-workflows/23d2ce059e3b2d00b1ca9fc2806280527bad31ae/w_smart-seq_clustering/scanpy_clustering_workflow.json)

In the case of the Galaxy workflow, the Galaxy instance needs to have installed the [suite_scanpy_scripts](http://toolshed.g2.bx.psu.edu/view/ebi-gxa/suite_scanpy_scripts) and [gtf2gene_list](https://toolshed.g2.bx.psu.edu/view/ebi-gxa/gtf2gene_list) Galaxy Toolshed packages. To run the clustering part you will need 10x matrix market format files and the GTF file for the organism in question.
