#### Data statistics

- Ensembl **104** / Ensembl Genomes **51** / WormBase ParaSite **15** gene annotations.   
  array design probe set mappings.   
- This release contains **4562** datasets (**161199** assays), in particular:            
  - **1512** RNA-seq and **123** proteomics datasets, of which **375** report
    [baseline](https://www.ebi.ac.uk/gxa/baseline/experiments) gene expression, across a total of **48** different
    organisms;           
  - **1026** datasets reporting expression in [plants](https://www.ebi.ac.uk/gxa/plant/experiments);               
  - **4187** datasets studying samples in **14587**
    [differential](https://www.ebi.ac.uk/gxa/experiments?experimentType=Differential) comparisons across **51**
    different organisms.
  - **67** overall species count

#### Software features

### New default view for baseline experiments
All baseline experiments - both transcriptomic and proteomic - now feature a **marker-gene-based view** as the default display. This new interface highlights the *top marker gene(s) per condition*, making it easier to explore tissue- or condition-specific expression patterns. 

Marker genes are identified using the Marker Gene Finder for RNA-seq (MGFR) algorithm, which assigns a specificity score to each gene based on how uniquely it is expressed. The complete marker gene lists for each experiment are available for download from our [FTP](https://ftp.ebi.ac.uk/pub/databases/microarray/data/atlas/experiments/).

Users can toggle between the *Marker Genes* and traditional expression views by selecting or deselecting the Marker Genes checkbox - for example, see [RNA-seq of coding RNA from seven Bos taurus tissues](https://www.ebi.ac.uk/gxa/experiments/E-MTAB-2596/Results). 

Further details are available in our [FAQ](https://www.ebi.ac.uk/gxa/FAQ.html) and [Help pages](https://www.ebi.ac.uk/gxa/help/index.html).

### Easier discovery of datasets by technology type
A new **technology filter** has been added to the experiment search page, making it easier to identify and browse  experiments by their technology (i.e. proteomics) and improving overall discoverability. 
